docsets/CUDA/docset.json

```diff
@@ -1,12 +1,12 @@
 {
     "name": "CUDA",
-    "version": "9.0",
+    "version": "9.2.88",
     "archive": "CUDA.tgz",
```

Comment:
You need to update the archive at this path as well.

Reply:
Ok, fixed.